Blazor/wwwroot/cdts-blazor.js

```diff
@@ -0,0 +1,234 @@
+            const url = new URL(cssHref);
+            const hostname = url.hostname.toLowerCase();
+            const pathname = url.pathname.toLowerCase();
+            const baseUrl = cssHref.substring(0, cssHref.lastIndexOf('/cdts/cdts-') + 1);
+
+            //theme
+            const theme = pathname.includes('/gcintranet/') ? 'gcintranet' : 'gcweb';
+
+            //cdnEnv
+            let cdnEnv = null;
+            if (hostname === 'www.canada.ca' || hostname === 'canada.ca') {
+                cdnEnv = 'prod';
+            }
+            else if (hostname === 'cdts.service.canada.ca') {
+                cdnEnv = theme === 'gcweb' ? 'esdcprod' : 'prod'; //depends on theme
+            }
+            else if (hostname === 'templates.service.gc.ca') {
+                cdnEnv = 'esdcprod';
+            }
+            else {
+                cdnEnv = baseUrl; //anything else is taken as-is minux the stylesheet's location, including trailing slash
+            }
+
+            this.cdnEnv = cdnEnv;
+
+            //version
+            let version = null;
+            if (pathname.includes('/rn/cls/') || pathname.includes('/rn/cdts/')) {
+                version = 'rn';
+            }
+            else {
+                version = (pathname.match(/\/(v[0-9]+_[0-9]+_[0-9]+)\//) || [null, null])[1];
+            }
+
+            return { cdnEnv, theme, version, baseUrl };
+        }
+        catch (error) {
+            console.error('Unexpected error parsing CDTS stylesheet URL, unable to derive CDTS parameters, defaults will be used.', error);
+            return null;
+        }
+    },
+
+    findCDTSCssHref: function findCDTSCssHref() {
+        return Array.from(document.head.querySelectorAll('link[rel="stylesheet"]')).map((e) => e.getAttribute('href')).find((href) => href?.includes('/cdts/cdts-')) || null;
+    },
+
+    resetExitScript: function resetExitScript(exitScriptObj) {
+        var elems = document.getElementsByTagName('a');
+        wet.utilities.wetExitScript(
+            exitScriptObj.displayModal != null ? exitScriptObj.displayModal.toString() : 'undefined',
+            exitScriptObj.exitURL != null ? exitScriptObj.exitURL : 'undefined',
+            exitScriptObj.exitDomains != null ? exitScriptObj.exitDomains : 'undefined',
+            exitScriptObj.exitMsg != null ? exitScriptObj.exitMsg : 'undefined',
+            exitScriptObj.yesMsg != null ? exitScriptObj.yesMsg : 'undefined',
+            exitScriptObj.cancelMsg != null ? exitScriptObj.cancelMsg : 'undefined',
+            exitScriptObj.msgBoxHeader != null ? exitScriptObj.msgBoxHeader : 'undefined',
+            exitScriptObj.targetWarning != null ? exitScriptObj.targetWarning : 'undefined',
+            exitScriptObj.displayModalForNewWindow != null ? exitScriptObj.displayModalForNewWindow.toString() : 'undefined',
+            elems);
+    },
+
+    resetExitScriptOnPage: function resetExitScriptOnPage() {
+        this.resetExitScript(this.exitScriptObj);
```

Comment:
As mentionned in previous comment, should consider possibility of object being null and not call resetExitScript if it is the case)